pkg/controller/argocd/argocd_controller.go

```diff
@@ -137,39 +110,20 @@ type ReconcileArgoCD struct {
 	scheme *runtime.Scheme
 }
 
-// Reconcile reads that state of the cluster for a ArgoCD object and makes changes based on the state read
-// and what is in the ArgoCD.Spec
+// Reconcile reads that state of the cluster for a ArgoCD Route object and makes changes based on the state read
+// and what is in the Route.Spec
 // Note:
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
 func (r *ReconcileArgoCD) Reconcile(request reconcile.Request) (reconcile.Result, error) {
```

Comment:
Should this be renamed, now that it's no longer reconciling ArgoCD objects?

Reply:
I've renamed it to `ReconcileArgoCDRoute`. I'm okay with `ReconcileRoute` as well